tests/phpunit/unit/Configuration/ResourceManagerTest.php

```diff
@@ -178,9 +178,9 @@ public function testDefaultRequest()
         $app = new Application(['resources' => $config]);
         $this->assertEquals('cli', $config->getRequest('protocol'));
         $this->assertEquals('bolt.dev', $config->getRequest('hostname'));
-        $this->assertEquals('http://bolt.dev/bolt', $config->getUrl('canonical'));
+        $this->assertEquals('cli://bolt.dev/bolt', $config->getUrl('canonical'));
         $this->assertEquals('cli://bolt.dev', $config->getUrl('host'));
-        $this->assertEquals('http://bolt.dev/', $config->getUrl('rooturl'));
+        $this->assertEquals('cli://bolt.dev/', $config->getUrl('rooturl'));
```

Comment:
No, not when I tested it on 3.3 but I'm gettings some differing results in https://github.com/bolt/bolt/pull/6209/files#diff-6eeac24760013e79aaf552a1ccb05ad9R236 (test.dev vs bolt.dev). I'll have a look.